htdocs/user/logout.php

```diff
@@ -99,6 +99,15 @@
 	$url .= (preg_match('/\?/', $url) ? '&' : '?').'dol_use_jmobile=1';
 }
 
+// Logout openid_connect sessions using OIDC logout URL if defined
+if (getDolGlobalInt('MAIN_MODULE_OPENIDCONNECT', 0) > 0 && !empty($_SESSION['OPENID_CONNECT']) && !empty($conf->global->MAIN_AUTHENTICATION_OIDC_LOGOUT_URL)) {
```

Comment:
!empty($conf->global->MAIN_AUTHENTICATION_OIDC_LOGOUT_URL)
is not allowed.
Can you replace with
getDolGlobalString('MAIN_AUTHENTICATION_OIDC_LOGOUT_URL') ?